columnar/src/columnar/merge/mod.rs

```diff
@@ -54,14 +52,34 @@ impl From<ColumnType> for ColumnTypeCategory {
     }
 }
 
+/// Merge several columnar table together.
+///
+/// If several columns with the same name are conflicting with the numerical types in the
```

Comment:
does that mean we can't have multiple types on a column, e.g. ip and i64

Reply:
Yes.
Only i64, u64, f64 cannot coexist.